helm/oauth2-proxy/README.md

```diff
@@ -159,6 +159,7 @@ Parameter | Description | Default
 `service.clusterIP` | cluster ip address | `nil`
 `service.loadBalancerIP` | ip of load balancer | `nil`
 `service.loadBalancerSourceRanges` | allowed source ranges in load balancer | `nil`
+`service.nodePortNumber` | external port number for the service when service.type is `NodePort` | `nil`
```

Comment:
why `nodePortNumber` instead of `nodePort`?

Reply:
Because there is already the property `service.portNumber` defined 5 lines above.

Reply:
Oh, that's ugly, actually! I would prefer to get rid of this one later on and introduce sane naming now, I think `nodePort` is both cleaner and closer to what the Helm community does.

Reply:
ok, done